docs/html-css/week-1/homework.md

```diff
@@ -4,28 +4,28 @@ title: Coursework
 sidebar_label: Coursework
 ---
 
-## 1) Advanced CSS Selectors
+## 1) HTML Reading (1 hour)
```

Comment:
I moved the `HTML Reading (1 hour)` section up here from below since it makes sense to start with HTML to mimic learning HTML first in the lesson.